README.adoc

```diff
@@ -45,6 +45,18 @@ Message[] msgs = folder.getMessages();
 This allows you to test both the sending side and receiving side.
 All you need to do is to drop the JAR in the classpath during the test.
 
+== Enforcing usage in tests
+
+This library works thanks to the https://jakarta.ee/specifications/mail/2.1/apidocs/jakarta.mail/jakarta/mail/session#getProvider(java.lang.String)[the JavaMail API] which allows to use different providers implementation to handle actual email sending.
+
+To ensure it's used in your tests, you need to define the following Java system properties:
+
+  - `mail.smtp.class` set to `org.jvnet.mock_javamail.MockTransport`
+  - `mail.pop3.class` set to `org.jvnet.mock_javamail.MockStore`
+  - `mail.imap.class` set to `org.jvnet.mock_javamail.MockStore`
+
+This can be done either manually or via a configuration in your test runner. For example, see https://maven.apache.org/surefire/maven-surefire-plugin/examples/system-properties.html[this page] for Maven.
```

Comment:
"Click here" or "this page" links are [discouraged](https://www.w3.org/QA/Tips/noClickHere). That web page explains how to write more descriptive link text.